core/drivers/gic.c

```diff
@@ -786,6 +805,26 @@ static void gic_it_raise_sgi(struct gic_data *gd __maybe_unused, size_t it,
 #endif
 }
 
+static void gic_it_release(struct gic_data *gd, size_t it)
+{
+	size_t idx = it / NUM_INTS_PER_REG;
+	uint32_t mask = 1 << (it % NUM_INTS_PER_REG);
+
+	/* Assigned to group0 */
```

Comment:
What about group1 interrupts?

Reply:
The API is used for change group0 interrupts to group1 interrupts, so if group1 interrupts call this API, it will be asserted